_component_design/checkboxes.md

```diff
@@ -218,17 +306,16 @@ In the case of an error, always include an error message below the checkbox grou
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-#### Indeterminate
+### Indeterminate
```

Comment:
we should prob treat this section more like we are for the switch in a table. Indeterminate checkboxes should only be used in a table and only for bulk selection.